### DISCUSSION TITLE
Read Guidelines

### DISCUSSION THREAD

**Discussion by OwenGlendower** [community/community#23825] [New to GitHub]
<p>I read the FAQ guide, but I got the “Read Guidelines” badge.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for the report <a class="mention" href="/owenglendower">@owenglendower</a>, let me take a look and see what happened there.  Thank you for being here!</p>

**Comment by OwenGlendower** [comment, marked as answer]
<p>Thank you, the badge appeared</p>
